modules/mongodb/src/test/java/org/testcontainers/containers/MongoDBContainerTest.java

```diff
@@ -9,22 +9,26 @@
 import com.mongodb.client.MongoClients;
 import com.mongodb.client.MongoCollection;
 import com.mongodb.client.TransactionBody;
+import org.assertj.core.api.Assertions;
 import org.bson.Document;
+import org.junit.Assert;
 import org.junit.Test;
 import org.testcontainers.utility.DockerImageName;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
unused import. Remove, please

Reply:
Doh. In my rush to get out the door I forgot to run those checks.  Will do when I get home.